otel-agent/ecs-ec2/config.yaml

```diff
@@ -63,20 +68,28 @@ exporters:
     application_name: "$APP_NAME"
     subsystem_name: "$SUB_SYS"
     application_name_attributes:
+    - "aws.ecs.cluster"
+    - "aws.ecs.cluster.name"
     - "aws.ecs.task.definition.family"
```

Comment:
should be "aws.ecs.task.family"
https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor#amazon-ecs